### DISCUSSION TITLE
Datadump с всички администрации в България

### DISCUSSION THREAD

**Discussion by kdichev** [XenBG/bulgarian-city-coat-of-arms#1] [General]
Може да ти е полезно (датата е взета от https://iisda.government.bg/)
https://raw.githubusercontent.com/kdichev/bg-gov-metadata/main/data.json

**Comment by XenBG** [comment]
А да знаеш дали има от къде да се вземе същата информация, но касаеща всеки град в България? Най-вече ме интересува списък с градовете, в който е налична и информация за официален сайт, пощенски код, телефони и мейл на общината.
